src/include/hardware.h

```diff
@@ -73,6 +73,23 @@
 #define WT_WRITE_ONCE(v, val) WT_RELEASE_WRITE_WITH_BARRIER(v, val)
 #endif
 
+#ifdef HAVE_NEOVERSE
+#define WT_ACQUIRE_READ(v, val)                                         \
+    do {                                                                \
+        if (sizeof((val)) == 1) {                                       \
+            asm volatile inline("ldaprb %w0, %1" : "=r"(v) : "Q"(val)); \
```

Comment:
style nit: Our other inline assembly calls use `__asm__ volatile` not `asm volatile inline`.
Do you know if `inline` makes a difference here? I can't find any docs that indicate whether it's useful or extraneous

Reply:
> This size is also used for inlining decisions. If you use asm inline instead of just asm, then for inlining purposes the size of the asm is taken as the minimum size, ignoring how many instructions GCC thinks it is. 

According to the GCC docs, I'll remove inline and see how it behaves.

Reply:
Interestingly:
>asm does not work with -std=c99

So `__asm__` is required for that version.

Reply:
The generated assembly was identical i'll remove the `inline`.

Reply:
3d2d06ba6